src/main/webapp/public/index.html

```diff
@@ -25,19 +25,10 @@
       Learn how to configure a non-root public URL by running `npm run build`.
```

Comment:
Same comment as the other PR. Let's make sure the authentication PR doesn't get removed when this one is merged.